Dockerfile

```diff
@@ -0,0 +1,21 @@
+#This is a docker file to generate a SQL server. It helps you to execute the integration tests and for development
+#For more information about the possibility for SQL server on docker, check this site out: https://hub.docker.com/_/microsoft-mssql-server 
+
+FROM mcr.microsoft.com/mssql/server:2019-latest 
+
+ENV ACCEPT_EULA=Y 
+ENV SA_PASSWORD=abcDEF123#
```

Comment:
Let's comment this line out and add a comment saying that they either need to uncomment & fill it in here or set the environment variable themselves. We shouldn't be checking in passwords (even temp ones) into the repo.

Reply:
Yeah you are absolutely right. What do you think of that?
```Dockerfile
#ENV SA_PASSWORD=ADD_YOUR_SA_PASSWORD
```
So it would be clear to change here the password for the sa user.